src/modules/UI/selectors.js

```diff
@@ -15,6 +16,19 @@ export const getWallets = (state: State) => {
   return wallets
 }
 
+export const getFioWallets = (state: State) => {
+  const { activeWalletIds = [] } = state.core.account
+  const fioWallets = []
```

Comment:
Returning a statically declared object will force a rerender for any mapStateToProps that use this selector.

Reply:
will be refactored